### PR TITLE
Implemented sending account name to backend get request

### PR DESCRIPTION
**Pull request by isose**
https://trello.com/c/h8RrnZms/24-submit-account-name-to-backend

### CODE REVIEW

File: client/src/LandingPage.js [hunk 1]
```diff
@@ -1,83 +1,104 @@
...
 
 function LandingPage() {
-  function submitAccountName() {
+  const [account, setAccount] = useState('');
+  const [character, setCharacter] = useState('');
+  const [pobCode, setPobCode] = useState('');
+
+  function handleSubmitAccount() {
+    axios
+      .get('http://localhost:3001/account/characters', {
```
Comment (AndyLiuCodes) [inline client/src/LandingPage.js]: Remove localhost

**Review by AndyLiuCodes — CHANGES_REQUESTED** [review]
After you remove the localhost, then it LGTM

**Review by AndyLiuCodes — APPROVED** [review]
LGTM

**Review by theninjawoohoo — APPROVED** [review]
No issues looks good to me

**Comment by isose** [comment]
[Submit account name to backend](https://trello.com/c/h8RrnZms/24-submit-account-name-to-backend)

**Pull request merged**